src/components/Header/Header.tsx

```diff
@@ -34,16 +35,29 @@ class Header extends UIComponent<any, any> {
 
     /** Align header content. */
     textAlign: PropTypes.oneOf(['left', 'center', 'right', 'justified']),
+
+    /** Custom values for styling variables. */
+    variables: PropTypes.object,
```

Comment:
Is it necessary to add the variables prop? We don't have it added in any of the other components.

Reply:
actually, this thing turns out to be necessary for the sake of preventing `variables` being passed further to the child DOM tree - otherwise we can end up with the following thing:

![image](https://user-images.githubusercontent.com/9024564/43903289-6dd01de6-9bec-11e8-9323-6b9ddba5ec7a.png)

For the sake of preventing this to happen the `variables` property has been declared as handled.

Reply:
Got it! We should address this in the other components as well. Approving this PR.